cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -59,6 +59,14 @@ set(ARROW_THIRDPARTY_DEPENDENCIES
     ZLIB
     ZSTD)
 
+# TODO(wesm): External GTest shared libraries are not currently
+# supported when building with MSVC because of the way that
+# conda-forge packages have 4 variants of the libraries packaged
+# together
+if(MSVC)
+  set(GTest_SOURCE "BUNDLED")
```

Comment:
Shouldn't we force that only if the user didn't pass `GTest_SOURCE` explicitly?

Reply:
Correct, will fix

Reply:
fixed